index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr Norman Borlaug</h1>
+      <p id="p">The man who saved a billion lives</p>
+      <figure id="img-div">
+        <img id="image" src="/Images/tribute-page-main-image.jpg" alt="image" />
+        <figcaption id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </figcaption>
+      </figure>
+       <div id="tribute-info">
```

Comment:
Please make sure you used semantics tags. Instead of using div here, you should use article because this section contains a group of information about the subject which should be placed in the article element for better semantics.
When we have a group of related content we put that in the article element.